vars/buildDotNetCore.groovy

```diff
@@ -93,7 +93,8 @@ void call(Map config=[:]) {
           withCredentials([
             string(credentialsId: 'github-auth-token', variable: 'gitToken')
           ]) {
-            release.trigger(tag, repoName, tag, gitToken)
+            def commitMessage = utils.getCommitMessage()
```

Comment:
Following @FayToward's PR then all files in the `var` folder should use correct types as opposed to `def`.  So this `def` should be `String`

Reply:
Done.